node/nodeTransactions_test.go

```diff
@@ -294,6 +296,97 @@ func TestNode_GetTransaction_ShouldNotFindAndReturnUnknown(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestNode_ComputeTransactionStatusAllBranches(t *testing.T) {
```

Comment:
Can remove the suffix "AllBranches" perhaps.

Reply:
done.